### PR TITLE
Travis: groom SCA jobs

### PR DESCRIPTION
**Pull request by keradus**
as we run stan, why not run PHP CS Fixer, which was already integrated ages ago ;)

### CODE REVIEW

File: .php_cs.dist [hunk 1]
```diff
@@ -57,7 +57,7 @@ return PhpCsFixer\Config::create()
             'no_empty_comment' => true,
             'no_empty_phpdoc' => true,
             'no_empty_statement' => true,
-            'no_extra_consecutive_blank_lines' => true,
+            'no_extra_blank_lines' => true,
```
Comment (keradus) [inline .php_cs.dist]: just a deprecation fix, no logical changes

File: .travis.yml [hunk 2]
```diff
@@ -54,10 +54,11 @@ jobs:
     - stage: Quality
       php: 7.2
       env: PHPStan
+      before_install:
+        - travis_retry composer require --dev --no-update friendsofphp/php-cs-fixer:^2.10 phpstan/phpstan:^0.9
       install: travis_retry composer update --prefer-dist --prefer-stable
-      before_script:
-        - travis_retry composer require --dev --prefer-dist --prefer-stable phpstan/phpstan:^0.9
```
Comment (keradus) [inline .travis.yml]: running `composer update` and `composer require {package}` was causing composer to create deps graph and conflicts twice, with `composer require --no-update {package}`, it only modify json file, and then resolve all deps/conflicts only one in `composer update` run

TIL: no change in logic, only performance grooming

**Comment by sebastianbergmann** [comment]
But `php-cs-fixer` is already part of the build: https://travis-ci.org/sebastianbergmann/phpunit/jobs/350952189

**Comment by keradus** [comment]
I'm so blind ! :dagger: 
thanks, i cleaned up the branch, still some small grooming that could be applied :D

**Comment by keradus** [comment]
@sebastianbergmann , would it make sense to you if we would switch PHP CS Fixer installed via composer ?

**Comment by keradus** [comment]
thanks for merging ;)

**Comment by sebastianbergmann** [comment]
I prefer using tools such as `php-cs-fixer` from a PHAR over installing them via Composer.

**Comment by keradus** [comment]
hm... currently, PHP CS Fixer is installed via phar, and phpstan via composer :D

**Comment by sebastianbergmann** [comment]
Because of phpstan's "weird" mix of static and dynamic analysis.

**Comment by keradus** [comment]
:|

thanks for reasoning ;)

**Pull request merged**